_ci/build.sh

```diff
@@ -8,7 +8,7 @@ pushd "$(dirname "${BASH_SOURCE[0]}")/../"
 source _ci/init_tag.sh
 
 # Change tag if you are on a branch
-if [ ! -z "$TRAVIS_BRANCH" -a "$TRAVIS_BRANCH" != "master" ]; then
```

Comment:
Why do we add TAG_NAME on master image?

Reply:
I use the same variable name on build.sh. Only for consistency.